scrapers/HelixStudios.yml

```diff
@@ -0,0 +1,82 @@
+name: "HelixStudios"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/video/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.helixstudios.com/model/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="text"]
+    scene:
+      Title: //title/text()
+      Date:
+        selector: //div[@class="info-items"]/span[@class="info-item date"]/text()
+        postProcess:
+          - replace:
+              - regex: th
+                with:
+              - regex: st
+                with:
+              - regex: rd
+                with:
+          - parseDate: Jan 02, 2006
+      Details:
+        selector: //div[@class="description-content less"]/p/text()
```

Comment:
this selector only gets the first paragraph from `long` descriptions and nothing at all for short ones
```yaml
        selector: //div[contains(@class,"description-content")]/p
        concat: "\n\n"
```
seems to work ok